### DISCUSSION TITLE
Default search location

### DISCUSSION THREAD

**Discussion by nexuspcs** [community/community#27241] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi, is it possible to changer the default search location when I use the search bar in the top left of a GitHub window?
For example, if I am on a repo's page, and choose to click the search bar in the top left, can I choose that the search bar will search all of GitHub by default, rather than just within that repo?

Thanks,

**Comment by ghost** [comment]
i think u can .. :")

**Comment by sawyerbivens** [comment, marked as answer]
@nexuspcs if you type something in the search bar it should give you options on where to search:

![image](https://user-images.githubusercontent.com/48366954/181160396-8b135466-45ab-40d4-ae81-253d55875e2d.png)
